code/game/gamemodes/changeling/powers/absorb.dm

```diff
@@ -105,3 +105,59 @@
 			protected_dna |= new_dna
 			return
 	absorbed_dna |= new_dna
+
+
+/obj/effect/proc_holder/changeling/swap_form
+	name = "Swap Forms"
+	desc = "We force ourselves into the body of another form, pushing their consciousness into the form we left behind."
+	helptext = "We will bring all our abilities with us, but we will lose our old form DNA in exchange for the new one. The process will seem suspicious to any observers."
+	chemical_cost = 40
+	dna_cost = 2
+	req_human = 1 //Monkeys can't grab
+	genetic_damage = 50
+
+/obj/effect/proc_holder/changeling/swap_form/can_sting(var/mob/living/carbon/user)
+	if(!..())
+		return
+	var/obj/item/weapon/grab/G = user.get_active_hand()
```

Comment:
Not entirely sure what you're on about here, exactly, or why it's even necessary.